pybind11/__main__.py

```diff
@@ -9,29 +9,34 @@
 
 
 def print_includes():
-    dirs = [sysconfig.get_path('include'),
-            sysconfig.get_path('platinclude'),
-            get_include()]
+    dirs = [
+        sysconfig.get_path("include"),
+        sysconfig.get_path("platinclude"),
+        get_include(),
+    ]
 
     # Make unique but preserve order
     unique_dirs = []
     for d in dirs:
         if d not in unique_dirs:
             unique_dirs.append(d)
 
-    print(' '.join('-I' + d for d in unique_dirs))
+    print(" ".join("-I" + d for d in unique_dirs))
```

Comment:
I mostly limited Black to just the new files; but it did affect the few files that didn't change much.

PS: Black makes flake8 _so_ much nicer. No more "you can't commit because you need two spaces here instead of one" errors. :)

Reply:
> I mostly limited Black to just the new files; but it did affect the few files that didn't change much.

Ah, right, OK, that explains things :-)

I tend to try and put too much of my thoughts into the styling, I think. So I don't like some program coming after me thinking it should clean up (at least not without me checking it and being able to overrule).

Reply:
You can always add format on/off commands, then you tell everyone you care about exactly how some bit of code is formatted. However, if it formats poorly, you can almost always rewrite your code to format better _and_ improve the code too. :)

Reply:
Let's see how things go ;-) :-)